Detectors/TRD/workflow/include/TRDWorkflow/VdAndExBCalibSpec.h

```diff
@@ -116,10 +123,13 @@ DataProcessorSpec getTRDVdAndExBCalibSpec()
   using device = o2::calibration::VdAndExBCalibDevice;
   using clbUtils = o2::calibration::Utils;
 
+  auto dataRequest = std::make_shared<o2::globaltracking::DataRequest>();
```

Comment:
I am only noticing this now. Why do you add the dataRequest here? It should not be needed at all I think. Neither is the recoContainer

Reply:
I think this was some dead code I forgot to delete...